.github/pull_request_template.md

```diff
@@ -9,8 +9,12 @@ See here for more information and a list of available options:
 http://tedana.readthedocs.io/en/latest/contributing.html#pull-requests
```

Comment:
Could you update this URL to point to the new CONTRIBUTING.md section on pull requests ? Otherwise this will return a 404 ! Sorry for missing it myself !!

Reply:
Done ✅